frontend/src/pages/NewProject/NewProject.module.css

```diff
@@ -24,9 +24,16 @@
 }
 
 .button {
+	width: 100%;
+	height: 30px;
 	display: flex;
-	margin-bottom: 0;
-	margin-top: 0;
+	flex-direction: row;
+	align-items: center;
+	max-width: max-content;
+	padding-top: 1px !important;
```

Comment:
Not a huge fan of having to use `!important`, but okay with it if necessary

Reply:
@SpennyNDaJets yeah this is required here